tools/tidb-binlog-kafka.md

```diff
@@ -117,52 +127,71 @@ zookeeper_addrs = "192.168.0.11:2181,192.168.0.12:2181,192.168.0.13:2181"
 
 ### Deploy Pump using Binary
 
+A usage example:
+
+For three PDs, three ZooKeepers, and one TiDB, the information of each node is as follows: 
+	
+```
+TiDB="192.168.0.10"
+PD1="192.168.0.16"
+PD2="192.168.0.15"
+PD3="192.168.0.14"
+ZK1="192.168.0.13"
+ZK2="192.168.0.12"
+ZK3="192.168.0.11"
+```
+
+Deploy Drainer Pump on the machine with the IP address "192.168.0.10".
+
+The IP address of the corresponding PD cluster is "192.168.0.16,192.168.0.15,192.168.0.14". 
+
+The ZooKeeper IP address of the corresponding Kafka cluster is "192.168.0.13,192.168.0.12,192.168.0.11".
+This example shows the Pump Drainer usage.
```

Comment:
Add a blank line before this sentence, and update it to:

This example describes how to use Pump/Drainer.